_docs-latest/query/syntax.md

```diff
@@ -0,0 +1,336 @@
+---
+title: "Query Syntax Guide"
+tags: [getting_started, query]
+summary: This page gives an overview of DataWave's JEXL and Lucene query syntax options
+---
+<h2>Introduction</h2>
+<p>DataWave will typically accept query expressions conforming to either JEXL syntax (the default) or a modified Lucene syntax.
+   As for JEXL, DataWave supports a subset of the language elements in the <a href="https://commons.apache.org/proper/commons-jexl/reference/syntax.html"
+   target="_blank">Apache Commons JEXL grammar</a> and also provides several custom JEXL functions. DataWave has
+   enabled support for Lucene expressions as a convenience and will provide equivalent functionality to JEXL,
+   except where noted below.
+</p>
+<h2>JEXL Query Syntax</h2>
+<hr />
+<h3>Supported JEXL Operators</h3>
+<ul>
+    <li>==</li>
+    <li>!=</li>
+    <li>&lt;</li>
+    <li>&le;</li>
+    <li>&gt;</li>
+    <li>&ge;</li>
+    <li>=~ (regex)</li>
+    <li>!~ (negative regex)</li>
+    <li>|| , or</li>
+    <li>&amp;&amp; , and</li>
+</ul>
+<hr />
+<h3>Custom JEXL Functions</h3>
+<ul>
+    <li>Content functions<ul><li>phrase()</li><li>adjacent()</li><li>within()</li></ul></li>
```

Comment:
I think we are missing the includeRegex and excludeRegex functions in here

Reply:
Nope. They're in there, under the 'Custom Lucene Functions' heading
http://localhost:4000/docs/2.1/query/syntax#custom-lucene-functions